client/blindai/client.py

```diff
@@ -280,6 +282,8 @@ class BlindAiClient:
     attestation: Optional[GetSgxQuoteWithCollateralReply] = None
     server_version: Optional[str] = None
     client_info: ClientInfo
+    tensor_inputs: Dict
```

Comment:
I don't think you should leave that here.

Reply:
Will correct that asap